.github/workflows/main.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+
+  build:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-latest, macOS-latest]
```

Comment:
Considering that `macOS` build consumes 10x times more free minutes than linux build, should we keep only windows and ubuntu one?

If we get rid of .NET 4.x at some point, I would remove even windows build as it will stop bringing any value. What do you think?

Reply:
At the moment we do not have issues with `free minutes` in `fsprojects` org.  I think that we can keep all 3 for now to increate test coverage and ensure that it is easy to start contributing from any platform. 

If we face issue with `free minutes` on org level, we will deal with it in the future (for example remove macOS from repos that consume largest amount of minutes)